CRM/Contribute/BAO/Contribution.php

```diff
@@ -5457,4 +5457,25 @@ public static function createProportionalFinancialEntries($entityParams, $lineIt
     }
   }
 
+  /**
+   * Calculate Tax for each item when Financial Type is changed.
+   *
+   * @param array $lineItem
+   *
```

Comment:
https://www.drupal.org/node/1354#functions suggests a blank line between args and no blank line after last one. fixed in updated PR.